release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -1860,6 +1860,11 @@ fail to boot. Do not select this option.
 * You might receive a message that the system is unable to mount a volume when the volume contains a large number of files. This can happen when a Pod mounts a volume that is set with `FSGroup SecurityContext` because the GID ownership of the files must be recursively updated for all files on the volume. Users should expect that Pods using volumes with a large number of files and the `FSGroup SecurityContext` setting can take considerable time to start.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1515907[*BZ#1515907*])
 
+* When running a node, a process leak can occur where conmon processes continue
+to spawn as zombie processes and never close. This is caused by conmon
+double-forking to daemonize and detach from the parent that launched it.
```

Comment:
I would change the wording here a bit:
```
Running pods with frequent probes can cause the number of conmon processes (a program which detaches from its parent: CRI-O, and is used to exec the container runtime) to grow quickly. If the probes happen frequently enough, systemd has trouble reaping all of its new children, and some conmon processes can become zombies.
```

basically, I'd focus on mentioning it's a problem with the frequency of probes, and that fewer probes or smaller workloads prevents this issue

Reply:
Thanks! Just updated!